### DISCUSSION TITLE
麻省理工學院機器豹 AI 學習新跑姿，跑步時速創 14 公里新紀錄

### DISCUSSION THREAD

**Discussion by luo910227** [1101-GS3073-C/Python-Community-GS3073-C#418] [科技新知]
「稍早研究人員開始使用 AI 驅動的模擬系統負責調整工作，訓練機器人因應不同場景調整步姿，讓機器豹成功突破，生成一種新跑姿，同時 Mini Cheeta 達到 14.04 公里跑速新紀錄。」

我覺得這個研究是很不容易的因為實際場景的變化本來就很多端，能把機器人用ai訓練到時速14公里以上已經很可貴。

資料來源：https://technews.tw/2022/03/21/how-the-mit-mini-cheetah-learns-to-run/

**Comment by AngusGuei** [comment]
我跟朋友最近一個 ａｉ讓麥塊裡的角色自動跑酷的功能，machine learning 後可以慢慢地走過學習過的跑酷道路，這就整整花了我們三天🥱

**Comment by YuHengChen54** [comment]
雖然是很有趣，不過機器人還是用輪子移動最快哈哈哈
